EIPS/eip-2024.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 2024
```

Comment:
It would be better naming this 131 following the original PR.

Reply:
Done